contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -826,6 +826,178 @@ $BODY$
 LANGUAGE plpgsql
 IMMUTABLE;
 
+
+CREATE OR REPLACE FUNCTION sys.SWITCHOFFSET(IN input_expr PG_CATALOG.TEXT,
```

Comment:
Why do we need `N tz_offset PG_CATALOG.TEXT` ? 
Is this `IN tz_offset anyelement` not sufficient? Can you give some examples?

Reply:
Here for datepart function we are doing the same as anyelement cannot handle type unknown.